ddtrace/contrib/trace_utils.py

```diff
@@ -595,3 +597,54 @@ def set_user(tracer, user_id, name=None, email=None, scope=None, role=None, sess
             "See https://docs.datadoghq.com/security_platform/application_security/setup_and_configure/"
             "?tab=set_user&code-lang=python for more information.",
         )
+
+
+def _start_span_internal(
+    tracer,  # type: Tracer
+    name,  # type: str
+    integration_config,  # type: IntegrationConfig
+    child_of=None,  # type: Optional[Union[Span, Context]]
+    service=None,  # type: Optional[str]
+    resource=None,  # type: Optional[str]
+    span_type=None,  # type: Optional[str]
+    activate=False,  # type: bool
+    meta=None,  # type: Optional[Dict[str, str]]
+    metrics=None,  # type: Optional[Dict[str, float]]
```

Comment:
How often will these parameters be used? We do not pass meta and metrics dict to `tracer._start_span()` since these are "internal" concepts. span.set_metric, span.set_metrics, span.set_tags() and span.set_str_tag() should be used instead. 

I am leaning towards removing these args